pkg/snapstore/utils.go

```diff
@@ -179,36 +178,37 @@ func adaptPrefix(snap *brtypes.Snapshot, snapstorePrefix string) string {
 	return snapstorePrefix
 }
 
-// GetSnapstoreSecretHash returns the hash of object store secrets hash
-func GetSnapstoreSecretHash(config *brtypes.SnapstoreConfig) (string, error) {
+// GetSnapstoreSecretModTime returns the latest modification timestamp of the access credential files.
+// Returns an error if fetching the timestamp of the access credential files fails.
+func GetSnapstoreSecretModTime(config *brtypes.SnapstoreConfig) (time.Time, error) {
 	switch config.Provider {
 	case brtypes.SnapstoreProviderLocal:
-		return "", nil
+		return time.Time{}, nil
 	case brtypes.SnapstoreProviderS3:
-		return S3SnapStoreHash(config)
+		return S3SnapStoreModTime(config)
 	case brtypes.SnapstoreProviderABS:
-		return ABSSnapStoreHash(config)
+		return ABSSnapStoreModTime(config)
 	case brtypes.SnapstoreProviderGCS:
-		return GCSSnapStoreHash(config)
+		return GCSSnapStoreModTime(config)
 	case brtypes.SnapstoreProviderSwift:
-		return SwiftSnapStoreHash(config)
+		return SwiftSnapStoreModTime(config)
 	case brtypes.SnapstoreProviderOSS:
-		return OSSSnapStoreHash(config)
+		return OSSSnapStoreModTime(config)
 	case brtypes.SnapstoreProviderOCS:
-		return OCSSnapStoreHash(config)
+		return OCSSnapStoreModTime(config)
 	default:
-		return "", nil
+		return time.Time{}, fmt.Errorf("Unknown provider: %v", config.Provider)
```

Comment:
It's a good practice in Go to start error messages with a lowercase letter, as these messages might be chained with others in different contexts.

```suggestion
		return time.Time{}, fmt.Errorf("unknown provider: %v", config.Provider)
```